pallets/btc-registration-pool/src/lib.rs

```diff
@@ -0,0 +1,159 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+mod pallet;
+pub mod weights;
+
+pub use pallet::pallet::*;
+use weights::WeightInfo;
+
+use sp_core::{ConstU32, RuntimeDebug};
+use sp_runtime::{BoundedBTreeMap, BoundedVec, DispatchError};
+use sp_std::vec::Vec;
+
+use parity_scale_codec::{Decode, Encode, MaxEncodedLen};
+use scale_info::TypeInfo;
+
+/// The maximum length of a valid Bitcoin address in characters (~62 alphanumeric characters).
+pub const ADDRESS_MAX_LENGTH: u32 = 62;
+
+/// The maximum amount of accounts a multi-sig account can consist.
+pub const MULTI_SIG_MAX_ACCOUNTS: u32 = 16;
```

Comment:
```suggestion
pub const MULTI_SIG_MAX_ACCOUNTS: u32 = 20;
```

정책상 16으로 정해진게 아니면, n은 20까지 지정될 수 있긴합니다.